content/docs/user-guide/action-manager/fundamentals/concepts/menus.md

```diff
@@ -0,0 +1,165 @@
+---
+linkTitle: Menus
+title: Menus
+description:
+weight: 102
+---
+
+## Menu
+
+The Menu Manager system provides interfaces to register and extend menus for use in the O3DE Editor UI.
+
+
+### Registration
+
+A new menu can be registered by just providing its string identifier.
+
+```
+
+AzToolsFramework::MenuProperties menuProperties;
+menuProperties.m_name = "Menu Name";
+
+menuManagerInterface->RegisterMenu(
+    "o3de.menu.identifier",
+    menuProperties
+);
+
+```
+
+The menu properties structure can also specify additional information:
+
+* A **name** string, which is meant to be a human-readable description to show in UI (currently unused);
+
+
+### Displaying the Menu
+
+Once a menu is registered, it is possible to retrieve their `QMenu*` and display them in the interface.
+
+```
+
+menuManagerInternalInterface->GetMenu("o3de.menu.identifier");
```

Comment:
```suggestion
QMenu* menu = menuManagerInternalInterface->GetMenu("o3de.menu.identifier");
```

Reply:
There's now a couple API calls to just display the Menu at a location, so I may just skip the GetMenu call in the docs. I'd rather add more formalized ways of accessing the functionality without passing through direct Qt raw pointers unless strictly necessary.